mne/_freesurfer.py

```diff
@@ -537,3 +539,62 @@ def _get_lut(fname=None):
     lut = {d[0]: np.array(lut[d[0]], dtype=d[1]) for d in dtype}
     assert len(lut['name']) > 0
     return lut
+
+
+def get_head_surface(surf, subject, subjects_dir, bem=None):
+    """Get a head surface from the Freesurfer subject directory.
+
+    Parameters
+    ----------
+    surf : str
+        The name of the surface 'auto', 'head', 'outer_skin', 'head-dense'
+        or 'seghead'.
+    %(subject)s
+    %(subjects_dir)s
+
+    Returns
+    -------
+    verts : ndarray
+        The surface vertices or None if ``surf='auto'`` and none is found.
+    triangles : ndarray
+        The vertex triangles or None if ``surf='auto'`` and none is found.
```

Comment:
The return value always appears to be a dict or None, is this outdated?

Reply:
After seeing all these doc errors, it might be cleaner just to remove the docstring altogether. Docstrings on private functions tend to get outdated rather quickly, so I tend not to use them (and often remove them on private functions) nowadays.